srcpkgs/linux5.13-surface/template

```diff
@@ -0,0 +1,303 @@
+# Template file for 'linux5.13-surface'
+pkgname=linux5.13-surface
+version=5.13.13
+revision=3
+wrksrc="linux-${version%.*}"
+short_desc="Linux kernel and modules containing patches and drivers for Microsoft Surface series devices ( series)"
+maintainer="Tino Heuberger <tinoheuberger@protonmail.com>"
+license="GPL-2.0-only"
+homepage="https://github.com/linux-surface/linux-surface"
+distfiles="https://cdn.kernel.org/pub/linux/kernel/v5.x/linux-${version%.*}.tar.xz
+ https://cdn.kernel.org/pub/linux/kernel/v5.x/patch-${version}.xz 
+ https://github.com/linux-surface/linux-surface/archive/refs/tags/arch-${version}-${revision}.tar.gz>surface-linux.tar.gz"
```

Comment:
Revision must be independent from any external resources, use version=5.13.13.3, `arch-${version%.*}-${version##*.}`

Reply:
Done, set revision to `1` and version to `5.13.13`
Using a static link to the linux-surface distfile